config/config.yaml

```diff
@@ -29,6 +29,10 @@ assets:
         rtl:
             basePath: '%assets_static_relative%/bootstrap-v4-rtl/dist/'
             link: css/bootstrap-rtl.min.css
+    # Used if we aren't using the theme or Bootstrap
+    bootstrap-js:
+        basePath: '%assets_static_relative%/bootstrap/dist/'
+        script: js/bootstrap.bundle.min.js
```

Comment:
There is already a backend mechanism to support this (with main themes, only bootstrap js is included). I'll look into extended this mechanism to the portal style call.
TODO for me